### PR TITLE
make builder and dman more efficient early on.

### PR DESCRIPTION
**Pull request by Raycoms**
Makes the colony a faster start

# Changes proposed in this pull request:
- Dman picks up 1 more stack each run
- Builder is twice as fast above y level 60.
-

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java [hunk 1]
```diff
@@ -38,6 +38,11 @@
      */
     private static final int STRENGTH_MULTIPLIER = 1;
 
+    /**
+     * At this y level the builder will be slower.
```
Comment (marchermans) [inline src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java]: slower?

**Review by marchermans — DISMISSED** [review]
Some small changes to the comments please but this works.

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count |  New  | Old  | Diff  |
| ------ |:-----:|:-----:|:----:|:-----:|
| Total  | 39048 | 14353 | 8389 | +5964 |
| Errors |  592  |  26   |  18  |  +8   |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=14595&tab=Inspection&buildTypeId=bt88)

**Comment by Raycoms** [comment]
Yeah, copy and paste error, added faster

**Review by LDTTeam-Buildsystem — APPROVED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 39048 |  0  |  0  |  0   |
| Errors |  592  |  0  |  0  |  0   |
##### More Information:

**Pull request merged**